Testscripts/Windows/CPU-Offline.ps1

```diff
@@ -197,37 +227,51 @@ install_package "fio"
 			$timeout = New-Timespan -Minutes 60
 			$sw = [diagnostics.stopwatch]::StartNew()
 			while ($sw.elapsed -lt $timeout){
-				$vmCount = $AllVMData.Count
 				Wait-Time -seconds 30
-				$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password -command "cat ~/state.txt"
+				$state = Run-LinuxCmd -ip $AllVMData[0].PublicIP -port $AllVMData[0].SSHPort -username $user -password $password "cat /home/$user/state.txt" -runAsSudo
+				Write-LogDbg "state is $state"
 				if ($state -eq "TestCompleted") {
-					$channelChangeCompleted = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password -command "cat ~/constants.sh | grep job_completed=0"
-					if ($channelChangeCompleted -ne "job_completed=0") {
-						throw "$local_script finished on $($VMData.RoleName) but job was not successful!"
+					$kernelCompileCompleted = Run-LinuxCmd -ip $AllVMData[0].PublicIP -port $AllVMData[0].SSHPort -username $user -password $password "cat /home/$user/constants.sh | grep job_completed=0" -runAsSudo
+					if ($kernelCompileCompleted -ne "job_completed=0") {
+						Write-LogErr "$local_script finished on $($AllVMData[0].RoleName) but setup was not successful!"
 					} else {
-						Write-LogInfo "$local_script finished on $($VMData.RoleName)"
-						$vmCount--
+						Write-LogInfo "$local_script finished on $($AllVMData[0].RoleName)"
+						break
 					}
-					break
 				} elseif ($state -eq "TestSkipped") {
+					Write-LogInfo "$local_script finished with SKIPPED state on $($AllVMData[0].RoleName)!"
 					$resultArr = $resultSkipped
-					throw "$local_script finished with SKIPPED state!"
+					$currentTestResult.TestResult = Get-FinalResultHeader -resultarr $resultArr
+					return $currentTestResult.TestResult
 				} elseif ($state -eq "TestFailed") {
+					Write-LogErr "$local_script finished the failed state on $($AllVMData[0].RoleName)."
```

Comment:
ditto

Reply:
Fixed in b59736da40b9a97a0de5d6b88cac572431e81fbc